assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -32,10 +37,22 @@ export default function useMediaPicker({
   type = '',
   multiple = false,
 }) {
+  const {
+    actions: { uploadVideoPoster },
+  } = useMedia();
   useEffect(() => {
     // Work around that forces default tab as upload tab.
     wp.media.controller.Library.prototype.defaults.contentUserSetting = false;
   });
+  useEffect(() => {
+    try {
+      wp.Uploader.prototype.success = ({ attributes }) => {
```

Comment:
Why not just extract the properties you need here.

Reply:
I wanted to do it, but then I saw `type` above... (`no-shadow`), so it's this vs renaming, no strong preference.

Reply:
I would rename the property. That is me.